catwalk/models/huggingface.py

```diff
@@ -42,7 +43,17 @@ def predict(  # type: ignore
             model = cached_transformers.get(AutoModelForQuestionAnswering, self.pretrained_model_name_or_path, False).to(device)
             tokenizer = cached_transformers.get_tokenizer(AutoTokenizer, self.pretrained_model_name_or_path)
             return self._predict_qa(qa_instances, model, tokenizer, batch_size=batch_size)
-        
+        elif task.has_instance_conversion(InstanceFormat.HF_CLASSIFICATION):
+            classification_instances = cast(
+                Sequence[HFClassificationInstance],
+                self._convert_instances(instances, InstanceFormat.HF_CLASSIFICATION, task))
+            model = cached_transformers.get(
+                AutoModelForSequenceClassification,
+                self.pretrained_model_name_or_path, False
```

Comment:
```suggestion
                self.pretrained_model_name_or_path, False,
                num_labels=task.num_labels
```

This suggestion needs `Task` to have an attribute saying how many labels it has. Something like this is required to run on anything other than the default `num_labels==2`

Reply:
done